README.md

```diff
@@ -101,6 +102,30 @@ $ fdiff [OPTIONS] [PRE-FONT FILE URL] [POST-FONT FILE FILE PATH]
 
 ⭐ **Tip**: Remote git repository hosting services (like Github) support access to files on different git branches by URL.  Use these repository branch URL to compare fonts across git branches in your repository.
 
+#### As Git's diff driver for fonts
+
+Git can be configured to automatically use a specific tool to diff specific file types. The are to steps to complete to use `fdiff` as the default diff output for font files.
+
+1. Tell Git how to run the tool. As with most git configuration options this may be set for a repository (`--local`), for your user (`--global`), or system (`--system`). We recommend setting this at the user level. Assuming `fdiff` is available in your path, this setting should do the trick:
+
+        git config --global diff.fdiff.command 'fdiff -c --git'
+
+    This will write it to your `$HOME/.gitconfig` file looking like this:
+
+    ```gitconfig
+    [diff "fdiff"]
+        command = fdiff -c --git
```

Comment:
Thinking aloud.  Is there a use case where we would not want the ANSI color formatted approach as part of the `--git` configuration?

Reply:
Maybe, I don't know. I set it here because most `git` setups I know of default to it, but I don't know how to make a dit driver inherit that setting from Git itself.

Reply:
SGTM